lib/tdf3/src/version.ts

```diff
@@ -1,4 +1,2 @@
-import { inBrowser } from './utils/index.js';
-
 export const version = '1.0.0';
```

Comment:
what is this version?  the TDF3 schema version?

Reply:
or package version?   can we bump the package version to 2.0.0

Reply:
It is the package version. See: https://github.com/opentdf/client-web/blob/main/scripts/bump-version.sh#L49